lib/src/internal/event_controller.dart

```diff
@@ -127,6 +127,14 @@ abstract class IWebsocketEventController implements IRestEventController {
   /// Emitted when a member joins a guild.
   Stream<IGuildMemberAddEvent> get onGuildMemberAdd;
 
+  /// Emitted when a member joins a guild but is not yet screened by:
+  /// https://support.discord.com/hc/en-us/articles/1500000466882
+  Stream<IGuildMemberAddEvent> get onGuildMemberAddScreening;
+
+  /// Emitted when a member joins a guild but passed member screening
+  /// https://support.discord.com/hc/en-us/articles/1500000466882
+  Stream<IGuildMemberAddEvent> get onGuildMemberAddPassedScreening;
```

Comment:
It should be an `IGuildMemberUpdateEvent` when user passed the screening

> When the member completes the screening, [Guild Member Update](https://discord.com/developers/docs/topics/gateway#guild-member-update) will be emitted and `pending` will be false.

Reply:
yeah, done :)